<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWidget</class>
 <widget class="QWidget" name="InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InputWidget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>591</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>QLineEidt</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>351</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>显示</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_clearBtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_readOnly">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>只读</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_inputH">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_inputV">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_echoMode">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>190</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_validator">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>191</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>水平方向</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>垂直方向</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>输入模式</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>验证码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_inputMask">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设置掩码</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>QTextEdit</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cursorWidth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lineWrapColumnOrWidth</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>tabStopWidth</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_cursorWidth">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_lineWrapColumnOrWidth">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_tabStopWidth">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_overwriteMode">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>183</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>overwriteMode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>lineWrapMode</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>256</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>wordWrapMode</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_tabChangesFocus">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>180</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>tabChangesFocus</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_lineWrapMode">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>220</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_wordWrapMode">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>491</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>QComboBox</string>
    </attribute>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>学生信息</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_student">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_editable">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>是否可以编辑</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="check_duplicatesEnabled">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>是否可以重名</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>最大显示数目</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>最大可录入的学生个数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>插入策略</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_maxCount">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_maxVisibleItems">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>170</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_insertPolicy">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_student">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>100</y>
       <width>171</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>当前学生信息</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>QSpinBox</string>
    </attribute>
    <widget class="WeekSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>321</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WeekSpinBox</class>
   <extends>QSpinBox</extends>
   <header>weekspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
